api/src/test/java/org/apache/iceberg/expressions/TestInclusiveManifestEvaluator.java

```diff
@@ -347,4 +349,52 @@ public void testStringStartsWith() {
     shouldRead = ManifestEvaluator.forRowFilter(startsWith("some_nulls", "1"), SPEC, false).eval(FILE);
     Assert.assertFalse("Should skip: range doesn't match", shouldRead);
   }
+
+  @Test
+  public void testIntegerIn() {
+    boolean shouldRead = ManifestEvaluator.forRowFilter(in("id", 5, 6), SPEC, true).eval(FILE);
+    Assert.assertFalse("Should not read: id below lower bound (5 < 30, 6 < 30)", shouldRead);
+
+    shouldRead = ManifestEvaluator.forRowFilter(in("id", 28, 29), SPEC, true).eval(FILE);
+    Assert.assertFalse("Should not read: id below lower bound (28 < 30, 29 < 30)", shouldRead);
+
+    shouldRead = ManifestEvaluator.forRowFilter(in("id", 30, 31), SPEC, true).eval(FILE);
```

Comment:
I think this test case should be `in("id", 29, 30)` to test the case where only one value is equal to or greater than the lower bound.

Reply:
Done.